lookout/style/format/rules.py

```diff
@@ -266,13 +268,15 @@ def __init__(self,
                                        number from 0 to 1 or the exact quantity.
         :param prune_attributes: indicates whether to remove useless parts of rules.
         :param uncertain_attributes: indicates whether to **retain** parts of rules with low \
-                                     certainty (see "Generating Production Rules From Decision
+                                     certainty (see "Generating Production Rules From Decision \
                                      Trees" by J.R. Quinlan).
+        :param prune_base_model: indicates whether to prune base_model via reduced error pruning \
+                                 algorithm. Available for DecisionTreeClassifier model only.
         """
         super().__init__()
+        self.prune_branches_algorithm = prune_branches_algorithm
         self.base_model = base_model
         self.prune_branches = prune_branches
-        self.prune_branches_algorithm = prune_branches_algorithm
```

Comment:
This swap of lines is not necessary: `self.prune_branches_algorithm = prune_branches_algorithm`